petab/lint.py

```diff
@@ -46,10 +69,17 @@ def check_parameter_df(df):
             f"Parameter table has wrong index {df.index.name}."
             "expected 'parameterId'.")
 
+    assert_presence_of_empty_spaces(df.index.values, "parameterId")
```

Comment:
Isn't that already included in the following loop?

Reply:
parameterId is not included in the `req_cols` list